tests/wpunit/AssetsTest.php

```diff
@@ -22,22 +22,53 @@ public function tearDown() {
 	/**
 	 * @test
 	 */
-	public function it_should_should_register_multiple_assets() {
+	public function it_should_register_multiple_assets() {
 		Asset::add( 'my-script', 'fake.js' )->register();
 		Asset::add( 'my-style', 'fake.css' )->register();
 
-		$this->assertTrue( Assets::init()->exists( 'my-script' ) );
-		$this->assertTrue( Assets::init()->exists( 'my-style' ) );
-		$this->assertTrue( wp_script_is( 'my-script', 'registered' ) );
-		$this->assertTrue( wp_style_is( 'my-style', 'registered' ) );
-		$this->assertEquals( 'my-script', Assets::init()->get( 'my-script' )->get_slug() );
-		$this->assertEquals( 'my-style', Assets::init()->get( 'my-style' )->get_slug() );
+		$this->existence_assertions( 'my' );
+	}
+
+	/**
+	 * @test
+	 */
+	public function it_should_locate_minified_versions_of_external_assets() {
+		if ( defined( SCRIPT_DEBUG ) ) {
```

Comment:
This backup of the `SCRIPT_DEBUG` constant must be moved to a `@before` method to make sure it will always happen when required.

Reply:
@lucatume i think using the `set_const_value` i dont need a `@before` anymore. Just an `@after` which should clear up any potential changes the test may have done and restore the constant to its original value.